### PR TITLE
Add Character.Ai as Easy

### PR DESCRIPTION
**Pull request by GuiReboucas**
# Character.ai (beta.character.ai)

https://beta.character.ai/settings/

Scroll to the bottom of the settings page on your profile and click ”Remove Account“, then insert your username and choose whether you want to export your data while deleting it or not.

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -20911,5 +20911,14 @@
         "domains": [
             "miyklas.com.ua"
         ]
+    },
+    {
```
Comment (tupaschoal) [inline _data/sites.json:20915]:
```suggestion
    },

    {
```

**Review by tupaschoal — COMMENTED** [review]
Can you move the entry so it is sorted on the JSON, please?

File: _data/sites.json [hunk 2]
```diff
@@ -20911,5 +20911,14 @@
...
+	"name": "character.ai",
+	"url": "https://beta.character.ai/settings/",
+	"difficulty": "easy",
+	"notes": "Scroll to the bottom of the settings page on your profile and click ”Remove Account“, then insert your username and choose whether you want to export your data while deleting it or not.",
+	"domains": [
+		"beta.character.ai"
```
Comment (Kefaku) [inline _data/sites.json:20921]:
```suggestion
		"character.ai",
		"book.character.ai",
		"beta.character.ai"
```

**Review by Kefaku — CHANGES_REQUESTED** [review]
There are 2 more domains in use:

**Review by tupaschoal — COMMENTED** [review]
The entry still needs to be moved to the right order in the JSON file.

**Comment by GuiReboucas** [comment]
> The entry still needs to be moved to the right order in the JSON file.

Oh damn my bad

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**